src/utils/parsers.ts

```diff
@@ -89,25 +89,18 @@ function parseSecretPathOptional(
   return [vaultName, directoryPath, value];
 }
```

Comment:
Currently, I am returning `undefined` if a secret path isn't provided, and secrets can detect that to perform a separate operation if needed. Is this approach fine, or should the parser itself return `'/'` if no secret path is provided?

Reply:
Have it return undefined since that part isn't actually provided. It's up to the commands themselves to decide how to handle it.